webpack/assets/javascripts/react_app/components/toastNotifications/toastListitem/Toast.consts.js

```diff
@@ -0,0 +1 @@
+export const defaultTimerDelay = 2000;
```

Comment:
according to http://www.patternfly.org/pattern-library/communication/toast-notifications/ this should be 8 seconds.